GameData/ROEngines/PartConfigs/Viking4_FRE.cfg

```diff
@@ -0,0 +1,79 @@
+PART
+{
+	module = Part
+	name = ROE-Viking4
+	author = Zarbizaure, capkirk
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+
+	MODEL
+	{
+		model = ROEngines/Assets/FRE/VikingUpper/VikingUpper
+	}
+
+	rescaleFactor = 1
+	attachRules = 1,1,1,0,0
+	node_stack_top = 0.0, 1.61, 0.0, 0.0, 1.0, 0.0
+	node_stack_bottom = 0.0, -2.1, 0.0, 0.0, -1.0, 0.0
+	node_attach = 0.0, 1.61, 0.0, 0.0, 1.0, 0.0
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.8 // engine nozzles are good at radiating.
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+
+	title = Viking (4/4B)
+	manufacturer = Snecma
+	description = Upper stage Viking engine used on Ariane 1 and 4 as well as on Indian Lauch Vehicles GSLV and PSLV. Includes configs for Viking 4, 4B, Vikas 2, 2B and X. [1.8m]
+
+	engineType = Viking
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
```

Comment:
Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?